packages/interbit-ui-tools/src/middleware/sagas/interbit.js

```diff
@@ -1,18 +1,22 @@
 const { call, put } = require('redux-saga/effects')
 
-const {
-  isConnected,
-  getInterbit,
-  waitForInterbit
-} = require('../interbitGlobal')
-
 const { actionCreators } = require('../actions')
 
-function* interbitContext() {
-  if (isConnected()) {
+function* getInterbitContext(runtimeContext) {
```

Comment:
This is where interbit is pulled from the runtimeContext and made available to the sagas.